test/mapmatch.cc

```diff
@@ -1428,10 +1421,50 @@ TEST(Mapmatch, test_breakage_distance_error_handling) {
   // If we get here then fail the test!
   FAIL() << "Expected trace_route breakage distance exceed exception was not found";
 }
+
+// Spot check OpenLR linear references when asked for
```

Comment:
imho these would be better off in a new gurka test designed specifically to verify these for both map matching and regular routes

Reply:
Yup